Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -48,7 +48,7 @@ final class CacheAdvanceTests: XCTestCase {
         let cache = try createCache(overwritesOldMessages: false)
         try cache.append(message: message)
 
-        let sameCache = try createCache(overwritesOldMessages: false, zeroOutExistingFile: false)
```

Comment:
We don't need this `zeroOutExistingFile` anymore since we only clear the cache explicitly with `clearCacheFile()`. Creating the cache with either of the `createCache()` variants won't clear it.